sqlflow_models/deep_embedding_cluster.py

```diff
@@ -84,14 +84,14 @@ def __init__(self,
         self._pretrain_epochs = pretrain_epochs
         self._pretrain_initializer = pretrain_initializer
         self._pretrain_lr = pretrain_lr
-        self._pretrain_optimizer = SGD(lr=self._pretrain_lr, momentum=0.9)
+        self._pretrain_optimizer = SGD(lr=float(self._pretrain_lr), momentum=0.9)
```

Comment:
The better way is that SQLFlow can derivate the float attribute value, I created an issue: https://github.com/sql-machine-learning/sqlflow/issues/910, we can follow it and remove the cast when the issue was closed.